StripeConnect/StripeConnectTests/WebView+Tests.swift

```diff
@@ -28,8 +28,8 @@ extension WKWebView {
                         """)
     }
 
-    func evaluateOnLoaderStart(elementTagName: String) {
-        evaluateMessage(name: "onSetterFunctionCalled",
+    func evaluateOnLoaderStart(elementTagName: String) async throws {
+        try await evaluateMessage(name: "onSetterFunctionCalled",
```

Comment:
Had to change this to an async method so we could wait for it to finish in `testJSOnLoadError`